app/uk/gov/hmrc/breathingspaceifproxy/connector/IndividualDetailsConnector.scala

```diff
@@ -16,44 +16,50 @@
 
 package uk.gov.hmrc.breathingspaceifproxy.connector
 
+import javax.inject.{Inject, Singleton}
+
 import scala.concurrent.ExecutionContext
 
 import cats.syntax.validated._
 import com.codahale.metrics.MetricRegistry
 import com.kenshoo.play.metrics.Metrics
-import javax.inject.{Inject, Singleton}
 import uk.gov.hmrc.breathingspaceifproxy._
 import uk.gov.hmrc.breathingspaceifproxy.config.AppConfig
 import uk.gov.hmrc.breathingspaceifproxy.metrics.HttpAPIMonitor
 import uk.gov.hmrc.breathingspaceifproxy.model._
 import uk.gov.hmrc.http._
+import uk.gov.hmrc.http.HttpReads.Implicits._
 
 @Singleton
-class DebtorDetailsConnector @Inject()(http: HttpClient, metrics: Metrics)(
+class IndividualDetailsConnector @Inject()(http: HttpClient, metrics: Metrics)(
```

Comment:
Should we not keep to our commonly agreed language on the project. These 'Individuals' are debtors in the breathing space context.

Reply:
Yes but our commonly agreed language at this point is `individual`. We have this everywhere in our documentation, even in the API doc.